### ISSUE TITLE
bump nwsapi to avoid global pollution?

### ISSUE THREAD

**Issue by cscheid** [b-fuze/deno-dom#151]
While linting the bundled releases of [quarto](https://quarto.org), we've noticed that deno-dom pollutes the global scope. We narrowed it down to [nwsapi.js:367, etc](https://github.com/b-fuze/deno-dom/blob/master/src/dom/selectors/nwsapi.js#L367).

The upstream [nwsapi](https://github.com/dperini/nwsapi/commit/69fdeae62574b0ee8b17510700c0274de16aebc8) library has fixes for these. Would updating `nwsapi` be considered a bug in scope in deno-dom?

**Comment by b-fuze** [comment]
Yes, we can do this. I've actually already prepared this, but put it off because of some fixes that the NWSApi developer said were resolved were not actually resolved when I tested them locally.

But if it pollutes the global namespace less then I'm all for it.
